src/main/scala/nl.knaw.dans.easy.license/LicenseCreatorApp.scala

```diff
@@ -33,21 +32,17 @@ class LicenseCreatorApp(configuration: Configuration) extends AutoCloseable {
     configuration.properties.getString("fsrdb.db-connection-username"),
     configuration.properties.getString("fsrdb.db-connection-password"))
   val fileLimit: Int = configuration.properties.getInt("license.fileLimit")
-  val ldapContext: InitialLdapContext = {
-    import java.{ util => ju }
-
-    val env = new ju.Hashtable[String, String]
+  val ldapEnv: LdapEnv = {
```

Comment:
I think you now should be able to do it in a cleaner way:

```scala
val ldapEnv: LdapEnv = new LdapEnv {
  put(Context.PROVIDER_URL, configuration.properties.getString("auth.ldap.url"))
  // etc.
}
```

Reply:
Correct.  Done.